code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -161,13 +162,30 @@
 	if (get_dist(src, user) <= 1)
 		checktime()
 
+/obj/item/clothing/gloves/watch/verb/changeformat()
+	set category = "Object"
+	set name = "Change Watch Format"
+	set src in usr
+
+	use_military_format = !use_military_format
+	to_chat(usr, "You toggle the format of your watch. Now it will display in [use_military_format ? "24" : "12"] hour time.")
+
 /obj/item/clothing/gloves/watch/verb/checktime()
 	set category = "Object"
 	set name = "Check Time"
 	set src in usr
 
+	var/modified_time
+	if(use_military_format)
+		modified_time = worldtime2text()
+	else
+		if(worldtime2hours() <= 12)
```

Comment:
Wouldn't this cause the watch to display 12 in the afternoon as 12 AM?

Reply:
That is a good point

Reply:
Will approve if you update this. You can simply do < 12 instead of <= 12.

Reply:
Though you may need to fiddle with the below else statement as well.

Reply:
Alright so I seemed to have made a mistake somewhere trying to add that commit. Now the branch this PR is on is out of date and it won't let me fetch origin.